doc/DeveloperGuide.md

```diff
@@ -94,6 +94,7 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. User interface should be intuitive enough for users who are not IT-savvy.
```

Comment:
Can you define `intuitive`? `IT-Savvy`?
Please be specific in listing the NFRs.
Also, try to identify more NFRs.